haiku-apps/benormal/benormal-1.0.recipe

```diff
@@ -0,0 +1,42 @@
+SUMMARY="An application that normalizes WAV files"
+DESCRIPTION="BeNormal, a command-line program designed to normalize WAV files."
+APP="BeNormal"
+HOMEPAGE="https://github.com/HaikuArchives/$APP"
+COPYRIGHT="2000 David Burnett"
+LICENSE="GNU GPL v2"
+REVISION="1"
+srcGitRev="8185622b237d790e589d92dc5ddf7bf58f5ccc2c"
+SOURCE_URI="$HOMEPAGE/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="1a199db29953dffbd15b61e614df5a5775842c4056e8d3f282ecbeb919f7bf8f"
+SOURCE_FILENAME="$APP-$portVersion-$srcGitRev.tar.gz"
+SOURCE_DIR="$APP-$srcGitRev"
+
+ARCHITECTURES="x86_gcc2 !x86_64"
+
+PROVIDES="
+	benormal = $portVersion
+	app:$APP = $portVersion
+	"
+REQUIRES="
+	haiku
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	"
+BUILD_PREREQUIRES="
+	makefile_engine
+	cmd:g++
+	cmd:make
+	"
+
+BUILD()
+{
+	make OBJ_DIR=objects $jobArgs
+}
+
+INSTALL()
+{
+	install -d $appsDir
```

Comment:
BeNormal does not even show a help when called with `-h` or `--help` so I think it would be nice to add the Readme.txt:
```diff
 INSTALL()
 {
-	install -d $appsDir
+	install -d $appsDir $docDir
 	install objects/$APP $appsDir
+	install Readme.txt $docDir
 }
```

Reply:
Thanks! Learned something every time. :)